tests/perf/service_invocation_http/service_invocation_http_test.go

```diff
@@ -183,6 +183,19 @@ func TestServiceInvocationHTTPPerformance(t *testing.T) {
 	t.Logf("dapr latency avg: %sms", fmt.Sprintf("%.2f", daprLatency))
 	t.Logf("added latency avg: %sms", fmt.Sprintf("%.2f", avg))
 
+	dapr_metrics:=utils.DaprMetrics{
```

Comment:
Please make sure to configure your editor to format the files on save. VS Code and other editors should be able to do that automatically.

For now, I've ran the formatter for you.

Reply:
Okay....
Thanks.